docs/pages/management/admin/labels.mdx

```diff
@@ -280,6 +282,58 @@ computer. Your Teleport user must be authorized to access the server.
    ip-192-168-13-57 ⟵ Tunnel       arch=x86_64,environment=dev,hostname=ip-192-168-13-57
    ```
 
+## Step 4/4. Apply ephemeral labels using `server_info`
+
+<Notice type="note">
+   Applying ephemeral labels is only supported for servers.
+</Notice>
```

Comment:
I would remove this from a `Notice` and place it as a paragraph before the "You can apply..." paragraph. This way, a reader will know what "ephemeral labels" refers to before they see the warning. Since this is near the top of the section heading, I don't think it needs a Notice.